### PR TITLE
Conway era

### PR DESCRIPTION
**Pull request by sjoerdvisscher**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense and have useful messages
    - [ ] Important changes are reflected in changelog.d of the affected packages
    - [ ] Relevant tickets are mentioned in commit messages
- PR
    - [ ] (For external contributions) Corresponding issue exists and is linked in the description
    - [ ] Targeting main unless this is a cherry-pick backport
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] If relevant, reference the ADR in the PR and reference the PR in the ADR
    - [ ] Reviewer requested

### CODE REVIEW

File: cardano-node-emulator/src/Cardano/Node/Emulator/Internal/Node/Params.hs [hunk 1]
```diff
@@ -178,10 +179,10 @@ instance Default Params where
   def = Params def (pParamsFromProtocolParams def) testnet emulatorEpochSize
 
 instance Default C.ProtocolParameters where
-  -- The protocol parameters as they are in the Alonzo era.
+  -- The protocol parameters as they are in the Conway era.
   def =
     C.ProtocolParameters
-      { protocolParamProtocolVersion = (8, 0)
+      { protocolParamProtocolVersion = (9, 0)
```
Comment (Jimbo4350) [inline cardano-node-emulator/src/Cardano/Node/Emulator/Internal/Node/Params.hs]: Should update  the comment to indicate PV9 is the bootstrap era

File: cabal.project [hunk 2]
```diff
@@ -47,5 +47,13 @@ package cardano-crypto-praos
   flags: -external-libsodium-vrf
 
 constraints:
-  cardano-api ^>= 8.36
-  , quickcheck-contractmodel ^>= 0.1.6
+  cardano-api ^>= 8.42
+
+source-repository-package
```
Comment (Jimbo4350) [inline cabal.project:52]: Why is an SRP needed for this?

**Review by Jimbo4350 — COMMENTED** [review]
This looks ok but I can't really approve it as I've never worked on the library. If you are happy with the changes I would just force merge it.

**Review by Jimbo4350 — APPROVED** [review]
This looks fine to me however I am approving with the caveat that I am not familiar with this library @sjoerdvisscher.

File: cabal.project (hunk 2 above)
Comment (sjoerdvisscher) [reply cabal.project:52]: Both this package and quickcheck-contractmodel hardcode the current era. So to keep making it possible to test Babbage era while Mainnet is still on it, the Conway code lives in a branch: https://github.com/input-output-hk/quickcheck-contractmodel/tree/Conway

When Mainnet switches to Conway, the Conway branch of quickcheck-contractmodel can be merged, a new version can be published on CHaP and the SRP can be removed.

**Pull request merged**